services/packages/packages.proto

```diff
@@ -92,14 +93,34 @@ message UpdateRequest {
 
 message UpdateReply { string debug_output = 1; }
 
-message ListInstalledRequest { PackageSystem package_system = 1; }
+
+message SearchRequest { 
```

Comment:
Add some documentation here, e.g. what happens if none of installed & available set, etc.